src/main/index.js

```diff
@@ -166,9 +166,9 @@ function runApp() {
   let mainWindow
   let startupUrl
 
-  app.commandLine.appendSwitch('enable-accelerated-video-decode')
-  app.commandLine.appendSwitch('enable-file-cookies')
-  app.commandLine.appendSwitch('ignore-gpu-blacklist')
+  // Enable hardware acceleration via VA-API
+  // https://chromium.googlesource.com/chromium/src/+/refs/heads/main/docs/gpu/vaapi.md
+  app.commandLine.appendSwitch('enable-features', 'VaapiVideoDecodeLinuxGL')
```

Comment:
As requested, here is the code with the operating system check (based on the Windows and macOS ones in the same file).
```suggestion
  if (process.platform === 'linux') {
    // Enable hardware acceleration via VA-API
    // https://chromium.googlesource.com/chromium/src/+/refs/heads/main/docs/gpu/vaapi.md
    app.commandLine.appendSwitch('enable-features', 'VaapiVideoDecodeLinuxGL')
  }
```

Reply:
> As requested, here is the code with the operating system check (based on the Windows and macOS ones in the same file).

`Win32` and `Darwin`? No wonder I couldn't find it...

I found [the documentation](https://nodejs.org/api/process.html#processplatform) for it, your code is correct.

Reply:
Just a tip for the future, if ever decide to contribute to FreeTube or another open source project again: Being rude to the maintainers of projects that you want to contribute to is not a great move.

Reply:
> Just a tip for the future, if ever decide to contribute to FreeTube or another open source project again: Being rude to the maintainers of projects that you want to contribute to is not a great move.

I don't think I was that rude. But yeah, I will try to be more polite.